### DISCUSSION TITLE
Is it possible to deploy this without webui/standalone?

### DISCUSSION THREAD

**Discussion by djnixy** [AlUlkesh/stable-diffusion-webui-images-browser#248] [Q&A]
Is it possible to deploy this without webui/standalone?

**Comment by AlUlkesh** [comment]
Depends on what your requirement is.

It cannot run without the webui code, since it uses some of that.

However I don't think it should be very difficult to make it run, without webui running.

What are you looking for exactly?

**Reply by djnixy** [reply]
I just want to use it as a separate image browser. I understand that most of functionality will be gone on standalone mode.

**Reply by AlUlkesh** [reply]
I looked into it in more detail, but unfortunately the webui code makes this less than straightforward.

If you are under Windows, in the past I've used this standalone browser: https://github.com/RupertAvery/DiffusionToolkit

Perhaps this meets your requirements?
